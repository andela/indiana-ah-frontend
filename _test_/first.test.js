import React from 'react';
import { shallow } from 'enzyme';

describe('first test', () => {
  const wrapper = shallow(<h1 className="heading">Author's Haven</h1>);
  it('should render a component', () => {
    expect(wrapper.length).toEqual(1);
    expect(wrapper.find('h1.heading')).toHaveLength(1);
  });
});
