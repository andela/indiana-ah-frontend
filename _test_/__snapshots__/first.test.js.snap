// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Button component renders correctly 1`] = `
<button
  className="sc-bdVaJa GOJIj"
>
  Submit
</button>
`;
